### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by AKeeganDev**
This branch features the implementation of REDUX. The branch includes the following additions to the build:

- /redux directory in the src/ directory
- /redux holds a js file for books (for books state)
- /redux holds a categories js file (for future feature - so far undecided on final functionality)
- /redux holds a configureStore.js file that holds the store configuration exported by the books.js and categories.js files
- both books & categories have a set of actions for the store to dispatch & there is a reducer that consumes the dispatch action

at this time, store is not configured, these files are purely for setup in preparation for their deployment later this week.

Thank you!

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (Meltrust) [inline src/redux/books/books.js]: - [ ] These action types need to follow the duck's pattern, according to the project requirements:

image


Hint:  Examples of duck's pattern would be: 

```javascript
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (Meltrust) [inline src/redux/categories/categories.js]: - [ ] These action types need to follow the duck's pattern, according to the project requirements:

image

Hint:  Examples of duck's pattern would be: 

```javascript
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AKeeganDev,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Great start!  Nice reducers.✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @AKeeganDev,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights 👏 
✅ Good pull request discription 👍 
✅ Professional README file 💯 
✅ App works as expected 🔥 

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**